lib/active_fedora/orders/association.rb

```diff
@@ -82,6 +82,12 @@ def delete_at(loc)
       target.delete_at(loc)
     end
 
+    # Delete all occurences of the specified target
+    # @param obj object to delete
+    def delete(obj)
```

Comment:
#delete_target is probably more accurate. I would expect #delete to delete the proxy, all the target stuff is just convenience.